### PR TITLE
[FLOSS T2] docs: added verification to the "Arrays" tutorial

### PR DESCRIPTION
**Pull request by sedadas**
Some key creation commands were missing automatic verification in the "Arrays" (docs/tutorials/arrays.md) tutorial. This PR adds this verification

There was no teamwork in this task as I don't have a teammate.

## Basics

These points need to be fulfilled for every PR:

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Review by flo91 — COMMENTED** [review]
**[FLOSS T2]**

Thanks for the PR!

That's a very useful addition to an existing tutorial test.
I tried running the `testmarkdown_array` testcase as well as `make run_all` and `kdb run_all` and all tests passed successfully. :+1:

```
Start 220: testshell_markdown_tutorial_arrays
236/274 Test #220: testshell_markdown_tutorial_arrays ............   Passed    0.70 sec
```

The second commit message doesn't contain a module-prefix as required by the PR description.

>  Details of what you changed are in commit messages
> (first line should have `module: short statement` syntax)

A even better approach would be to amend to second commit to the first commit, because it only fixes two small typos.

There is currently one failing check at the buildserver about a changed manpage, but rebasing the PR with current master should probably solve that issue.

File: doc/tutorials/arrays.md [hunk 1]
```diff
@@ -62,9 +62,11 @@ To create an **array element** we start the basename of a key with the `#` chara
 
 ```sh
 kdb set user:/tests/sequence/#0 'First Element'
+#> Create a new key user:/tests/sequence/#0 with string "First Element"
 kdb set user:/tests/sequence/#1 'Second Element'
-# Arrays do not need to be contiguous
```
Comment (markus2330) [inline doc/tutorials/arrays.md:66]: Why do you remove this comment? It seems useful to me.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Is it really needed to check for the exact output of these commands? Does the successful return code not already tell enough?

File: doc/tutorials/arrays.md (hunk 1 above)
Comment (sedadas) [reply doc/tutorials/arrays.md:66]: I must have removed it unintentionally. I'll bring it back

**Comment by sedadas** [comment]
> Is it really needed to check for the exact output of these commands? Does the successful return code not already tell enough?

I did it in order to be consistent with the other examples in the page. The return code should also be enough, though

**Review by markus2330 — APPROVED** [review]
Thank you for the PR!

**Comment by markus2330** [comment]
Thank you for the PR!

**Pull request merged**